### ISSUE TITLE
Markdown standard

### ISSUE THREAD

**Issue by plafue** [jffrymrtn/writeily-pro#101]
Based on #99 we should clearly define which markdown standard the app supports. Would be helpful for us and the users

**Comment by jffrymrtn** [comment]
Definitely. Good idea -- my initial goal was to support Github flavoured Markdown, but I think that level of Markdown is out of reach (they have a lot of modifications on top of regular MD). So, I'd say, for starters, why not go with standard Markdown, including:

- Headers (#, ---, ===)
- Bold, Italics, Underline, the usual (*, _)
- Code blocks (via indented text)
- Tables
- Images and URLs (AFAIK these are supported by default anyway, like bold italics and underline)

I think all we're missing is code blocks and tables...

**Comment by plafue** [comment]
I just checked anddown's code, which is a wrapper for hoedown. Hoedown itself has tests against the "Markdown v1.0.3" specification (whatever that means). Their test suite is [here](https://github.com/hoedown/hoedown/tree/master/test/MarkdownTest_1.0.3). The test suite points to John Gruber's [original post](http://daringfireball.net/projects/markdown/).

**Comment by plafue** [comment]
I think it's your call, Jeff. I'm up for anything, as long as we keep it as much standards compliant (and free as in freedom) as possible.

**Comment by plafue** [comment]
> I think all we're missing is code blocks and tables...
By that you mean anddown?

**Comment by plafue** [comment]
I just checked out #82 - actually the syntax highlighting that i created some days ago differs a lot from the rendered version that you could get either from anddown or txtmark.

**Comment by plafue** [comment]
I will revisit this soon, together with the in-app markdown help
